packages/firebase_vertexai/firebase_vertexai/lib/src/api.dart

```diff
@@ -722,6 +763,8 @@ final class GenerationConfig extends BaseGenerationConfig {
     super.temperature,
     super.topP,
     super.topK,
+    super.presencePenalty,
```

Comment:
LiveGenerativeConfig would also need this two super

Reply:
Good call. Done.